release/bump_version_in_main.sh

```diff
@@ -109,7 +141,7 @@ echo "Created new branch: $branch_name"
 
 # Commit the changes
 git add .
-commit_message="build(release): bump version to $new_version"
+commit_message="build(release): bump version to $new_rust_version (Rust) and $new_cpp_version (C++)"
```

Comment:
this can be reverted, we don't distinguish rust and c++ versions. just that cmakelist.txt can't accept -dev suffix